zk.js/src/test-utils/testTransaction.ts

```diff
@@ -273,81 +273,6 @@ export class TestTransaction {
         connection: this.provider.provider!.connection,
       });
     }
-    let leavesAccountData;
```

Comment:
Where do we check that the leaves were inserted now?
Also why not keep this testing code around for the legacy tests?

Reply:
I hope I addressed it with https://github.com/Lightprotocol/light-protocol/pull/367/commits/10c3ff83d361bdb0e211e413b643634f92b8d255